src/lsst/cm/tools/cli/commands.py

```diff
@@ -300,11 +300,12 @@ def check(dbi: DbInterface, script_method: ScriptMethod, **kwargs: Any) -> None:
 @options.group()
 @options.workflow()
 @options.script_method()
-def accept(dbi: DbInterface, script_method: ScriptMethod, **kwargs: Any) -> None:
+@options.rescuable()
+def accept(dbi: DbInterface, script_method: ScriptMethod, rescuable=False, **kwargs: Any) -> None:
```

Comment:
Missing "bool" type hint on this one.  Also, might be better to set the default once, in the option declaration, since all of these seem to default to False?